overlays/crypto/blst.nix

```diff
@@ -10,10 +11,45 @@ stdenv.mkDerivation rec {
     sha256 = "01m28xxmm1x7riwyax7v9rycwl5isi06h2b2hl4gxnnylkayisn5";
   };
 
-  buildPhase = "./build.sh";
+  buildPhase = ''
+    ./build.sh ${lib.optionalString stdenv.targetPlatform.isWindows "flavour=mingw64"}
+  '' + ''
+    ./build.sh -shared ${lib.optionalString stdenv.targetPlatform.isWindows "flavour=mingw64"}
+  '';
   installPhase = ''
     mkdir -p $out/lib
-    cp ./libblst.a $out/lib/
+    for lib in libblst.{a,so,dylib}; do
+      if [ -f $lib ]; then
+        cp $lib $out/lib/
+      fi
+    done
+    for lib in blst.dll; do
+      if [ -f $lib ]; then
+        mkdir -p $out/bin
+        cp $lib $out/bin/
+      fi
+    done
+  '' + ''
+    mkdir -p $out/lib/pkgconfig
+    cat <<EOF > $out/lib/pkgconfig/libblst.pc
+    prefix=$out
+    exec_prefix=''\\''${prefix}
+    libdir=$out/lib
+
+    Name: libblst
+    Version: ${version}
+    Description: ${meta.description}
+
+    Libs: -L''\\''${libdir} -lblst
+    EOF
+  '';
```

Comment:
How much of this is done by `installPhase` already? could you get away with adding something to `postInstallPhase` (e.g. the pkgconfig stuff?).

Reply:
Nothing really. There is zero install logic in the repo. All they have is this very basic setup.sh script: https://github.com/supranational/blst/blob/master/build.sh